site_scons/site_tools/subst.py

```diff
@@ -20,8 +20,8 @@
 # Do the substitution
 def _subst_file(target, source, env, pattern, replace):
     # Read file
-    #print 'CALLING SUBST_FILE'
-    f = open(source, "rU")
+    # print 'CALLING SUBST_FILE'
+    f = open(source, "r")
```

Comment:
Rather than this whole bonkers thing which is a holdover from Python 2 days, let's just do

```
contents = Path(source).read_text()
```